tools/jenkins/run_jenkins.sh

```diff
@@ -89,8 +89,9 @@ then
     bash -l /var/local/jenkins/grpc/tools/jenkins/docker_run_jenkins.sh || DOCKER_FAILED="true"
 
   DOCKER_CID=`cat docker.cid`
-  docker kill $DOCKER_CID
+  docker kill $DOCKER_CID || true
```

Comment:
Can I suggest adding a comment about the above? Otherwise, it looks like a case of "This is fine"